import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/StorageCommitmentService.java

```diff
@@ -0,0 +1,251 @@
+package com.google.cloud.healthcare.imaging.dicomadapter;
```

Comment:
Please copy and paste the legal disclaimer at the top of this file. (we can add it to other files which have it missing in a follow up PR)

Reply:
Ok